src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/config/acl.yml

```
@@ -121,3 +121,8 @@ pim_enrich_job_tracker_view_all_jobs:
     label: pim_enrich.acl.job_tracker.view_all_jobs
     group_name: pim_enrich.acl_group.job_tracker
     enabled_at_creation: false
+
+pim_importexport_import_stop_job:
```

```suggestion
pim_importexport_stop_job:
```

Also changed in src/Akeneo/Platform/Bundle/ImportExportBundle/Normalizer/JobExecutionNormalizer.php and in src/Akeneo/Platform/Bundle/ImportExportBundle/Datagrid/JobExecutionResultRecordHydrator.php